content/200-concepts/200-database-connectors/08-cockroachdb.mdx

```diff
@@ -0,0 +1,71 @@
+  url      = env("DATABASE_URL")
+}
+```
+
+The fields passed to the `datasource` block are:
+
+- `provider`: Specifies the `cockroachdb` data source connector.
+- `url`: Specifies the [connection URL](#connection-details) for the CockroachDB database server. In this case, an [environment variable is used](/concepts/components/prisma-schema#accessing-environment-variables-from-the-schema) to provide the connection URL.
+
+Because CockroachDB support is currently a preview feature, you also need to include it in the list of preview features in the [`generator`]() block:
+
+```prisma file=schema.prisma
+generator client {
+  provider        = "prisma-client-js"
+  previewFeatures = ["cockroachdb"]
+}
+```
+
+## Connection details
+
+CockroachDB uses the PostgreSQL format for its connection URL. See the [PostgreSQL connector documentation](/concepts/database-connectors/postgresql#connection-details) for details of this format, and the optional arguments it takes.
+
+## Type mapping between CockroachDB and the Prisma schema
+
+The CockroachDB connector maps the [scalar types](/concepts/components/prisma-schema/data-model#scalar-fields) from the Prisma [data model](/concepts/components/prisma-schema/data-model) to native column types. These native types are mostly the same as for PostgreSQL — see the [PostgreSQL connector documentation](/concepts/database-connectors/postgresql#type-mapping-between-postgresql-to-prisma-schema) for details. However, there are some limitations, listed below.
+
+> Alternatively, see the [Prisma schema
+> reference](/reference/api-reference/prisma-schema-reference#model-field-scalar-types)
+> for type mappings organized by Prisma type.
+
+### Type mapping limitations
+
+When introspecting a CockroachDB database, the database types are mapped to Prisma in the same way as for PostgreSQL, with the following limitations:
+
+| CockroachDB (Type \| Aliases) | Prisma    | Supported | Native database type attribute | Notes                                                                                                                      |
+| ----------------------------- | --------- | :-------: | :----------------------------- | :------------------------------------------------------------------------------------------------------------------------- |
+| `money`                       | `Decimal` |  Not yet  | `@db.Money`                    | Supported in PostgreSQL but [not currently in CockroachDB](https://github.com/cockroachdb/cockroach/issues/41578)          |
+| `xml`                         | `String`  |  Not yet  | `@db.Xml`                      | Supported in PostgreSQL but [not currently in CockroachDB](https://github.com/cockroachdb/cockroach/issues/43355)          |
+| `jsonb` arrays                | `Json[]`  |  Not yet  | N/A                            | `Json[]` supported in PostgreSQL but [not currently in CockroachDB](https://github.com/cockroachdb/cockroach/issues/23468) |
+
+## Other limitations
```

Comment:
We might have something else (based on what we learned via your work on the tutorial) for this list. It is more a "difference" though, not a "limitation" - something just work different because of the default configuration of CockraochDB. Should we add a new section `## Differences` or put it together with the limitations somehow?

Reply:
I've added it to a new 'Differences' section for now

Reply:
Happy for comments both on organisation and on content - does this need more information?